lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -800,9 +800,7 @@ class ConvertAtenLayerNormOp : public OpConversionPattern<AtenLayerNormOp> {
 
     // Calculate squareSum for the layer.
     SmallVector<AffineMap> squareSumIndexingMaps{
-        inputShapeAffineMap,
```

Comment:
Did you use `clang-format` to format the code? Seems `clang-format` suggested formatting as the original code.